rero_ils/modules/utils.py

```diff
@@ -419,3 +421,43 @@ def get_patron_from_arguments(**kwargs):
         or Patron.get_patron_by_barcode(kwargs.get('patron_barcode')) \
         or Patron.get_record_by_pid(kwargs.get('patron_pid')) \
         or Patron.get_record_by_pid(kwargs.get('loan').get('patron_pid'))
+
+
+def set_timestamp(name, **kwargs):
+    """Set timestamp in current cache.
```

Comment:
Can you give more explanation? What is the purpose of this?

Reply:
added:
```
    Allows to timestamp functionality and monitoring of the changed
    timestamps externaly via url requests.
```